device/bgpcapability.yaml

```diff
@@ -0,0 +1,133 @@
+components:
+  schemas:
+    Device.BgpCapability:
+      x-status: under-review
+      description: >-
+        Container for BGP capability settings.
+      type: object
+      properties:
+        vpls:
+          description: >-
+            TBD
+          type: boolean
+          default: false
+        route_refresh:
+          description: >-
+            TBD
+          type: boolean
+          default: false
```

Comment:
Here are the defaults Google currently uses internally. I don't have a record of how we arrived at these, but thought it would be a useful datapoint:

	Ipv4Unicast:   true,
	Ipv4Multicast: true,
	Ipv4MplsVpn:   true,
	Ipv6Unicast:   true,
	Ipv6Multicast: true,
	Ipv6MplsVpn:   true,
	Vpls:          true,
	RouteRefresh:  true,